src/store/selectors/index.js

```diff
@@ -0,0 +1,42 @@
+'use babel'
+
+import { createSelector } from 'reselect'
+import ruleDocURIs from 'eslint-rule-documentation'
+import r from '../../f-utils/mini-ramda'
+import { deepProp } from '../../f-utils'
+import * as rules from './rules'
+import diffDict from '../../f-utils/deep-diff-dict'
```

Comment:
This is a similar story to `ruleDocURIs`, but while both of these _never_ change, this at least might have the possibility of changing in the future so I'm not quite as strongly feeling that this _needs_ to be moved into `rules.js`, I'd definitely _prefer_ it though as it makes it clearer where this is actually needed.

Reply:
same, response. If we move the other one, this should go too though.